src/typeInference/annotateFunctionCall.ts

```diff
@@ -62,6 +62,8 @@ export function annotateFunctionCall(
 	}
 
 	// If we found a return type, we annotate the AST with it
-	astHelper.insertAttribute(ast, 'type', functionProps.returnType);
+	if (functionProps.returnType.type !== ValueType.ITEM) {
```

Comment:
But is `item()?` not valuable to output? The cardinality of the sequence can be useful for optimizations: for `item()+` you can for example remove any checks for empty sequences.

Reply:
We are using `item()` as some sort of escape hatch, so to cover `item()+` would introduce a lot of problems for now